ai-quick-actions/model-deployment-tips.md

```diff
@@ -85,7 +94,7 @@ body = {
     "top_p": 0.8,
 }
 
-res = requests.post(endpoint, json=body, auth=signer, headers={}).json()
+res = requests.post(endpoint, json=body, auth=auth, headers={}).json()
```

Comment:
it should have been auth instead of signer. Originally, we had `signer = oci.auth.signers.SecurityTokenSigner(token, private_key)` but there was an update to this page last week which changed signer to auth, but this line wasn't updated.